### ISSUE TITLE
Syntax highlighting not functioning in latest version

### ISSUE THREAD

**Issue by danyill** [asciidoctor/asciidoctor-vscode#207]
## Description

Syntax highlighting has disappeared 

look at `exthost.log` I see:

```
[2019-06-18 11:57:57.549] [exthost] [error] [joaompinto.asciidoctor-vscode] provider FAILED
[2019-06-18 11:57:57.550] [exthost] [error] TypeError: tokens.filter is not a function
    at AsciidocFoldingProvider.<anonymous> (/home/mulhollandd/.vscode/extensions/joaompinto.asciidoctor-vscode-2.7.6/src/features/foldingProvider.ts:28:32)
    at Generator.next (<anonymous>)
    at fulfilled (/home/mulhollandd/.vscode/extensions/joaompinto.asciidoctor-vscode-2.7.6/out/src/features/foldingProvider.js:8:58)
```

## System Information

```
$ code --version
1.35.1
c7d83e57cd18f18026a8162d042843bda1bcf21f
x64

$ uname -a
Linux gerbra 5.1.0-050100-generic #201905052130 SMP Mon May 6 01:32:59 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.2 LTS
Release:        18.04
Codename:       bionic
```

Using extension AsciiDoc 2.7.6

## To Reproduce

Just open an Asciidoc file.

**Comment by danyill** [comment]
Using extension 2.7.6 on Windows 10 and VS Code version 1.35.1 appears to work fine so I'm not sure if the problem is related to my environment.

**Comment by dave986** [comment]
I have the same problem as @danyill, so, no source highlighting in the side preview, however highlighting works in the editor (not preview) when using triple backticks blocks:
````
```js
var a = 10;
myfunc(a);
// This is colorized in the editor, but not in the preview :'(
```
````

 I got the same exthost.log errors:
`
 [2019-08-20 15:44:59.728] [exthost] [error] [joaompinto.asciidoctor-vscode] provider FAILED
 [2019-08-20 15:44:59.728] [exthost] [error] TypeError: tokens.filter is not a function
	at AsciidocFoldingProvider.<anonymous> (/home/david/.vscode/extensions/joaompinto.asciidoctor-vscode-2.7.6/out/src/features/foldingProvider.js:29:42)
	at Generator.next (<anonymous>)
	at fulfilled (/home/david/.vscode/extensions/joaompinto.asciidoctor-vscode-2.7.6/out/src/features/foldingProvider.js:8:58)` 

My env:
`code --version
1.37.1
f06011ac164ae4dc8e753a3fe7f9549844d15e35
x64
`
`uname -a
Linux davidtp 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
`
`lsb_release -a
No LSB modules are available.
Distributor ID:	LinuxMint
Description:	Linux Mint 19.1 Tessa
Release:	19.1
Codename:	tessa
`
`xfce4-about -V
xfce4-about 4.13.4 (Xfce 4.12)`
AsciiDoc plugin: v2.7.6

I also tried to install nearly all older plugin versions and the highlighting does not work in either of them. This does not seem to be language-specific, it is the same for js, python, ruby...

What might be wrong?
